.travis/test/smoke

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+docker-compose --file docker-compose.yaml up -d web
```

Comment:
Should stop this so it's not running during the subsequent checks?

Reply:
It does no harm as it's separate containers, but better to clean up in case debugging output is displayed